roles/backpack_kube/tasks/main.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: Create Backpack Namespace
+  k8s: 
+    name: backpack
+    api_version: v1
+    kind: Namespace
+    state: present
+    kubeconfig: "{{ kube_config }}"
+
+- name: Launch backpack daemonset
+  k8s:
+    state: present
+    definition: "{{ lookup('file', 'roles/backpack_kube/files/backpack_set.yaml') }}"
+    kubeconfig: "{{ kube_config }}"
+  register: my_daemon_set
+
+- name: Sleep 5 to allow process time
+  pause:
+    seconds: 5
+
+- name: Wait until all pods exit creating state
+  k8s_facts:
+    kind: Pod
+    kubeconfig: "{{ kube_config }}"
+    label_selectors:
+      - name = backpack
+  register: pod_list
+  until: "'Creating' not in pod_list | json_query('resources[].status.phase')"
+
+
+- name: Wait for all pods to contain BACKPACK_DONE
+  shell: "/usr/local/bin/kubectl --kubeconfig={{ kube_config }} -n backpack logs -l name=backpack | grep BACKPACK_DONE | wc -l"
+  register: done_count
+  until: "done_count.stdout|int == pod_list.resources|length"
+  retries: 60
+  delay: 10
+
+- name: Create temporary directory
+  file:
```

Comment:
Can we call it backpack_output_dir instead? as this set of files would still be available after the run for the user to take a look at it

Reply:
Updated